tests/trainer/flags/test_min_max_epochs.py

```diff
@@ -41,3 +43,11 @@ def test_max_epochs_not_set_warning():
     with pytest.warns(PossibleUserWarning, match="`max_epochs` was not set. Setting it to 1000 epochs."):
         trainer = Trainer(max_epochs=None)
         assert trainer.max_epochs == 1000
+
+
+def test_fast_dev_run_no_warning():
+    """Test that no warning is emitted when `fast_dev_run` is set."""
+    with warnings.catch_warnings():
+        warnings.simplefilter("error")
```

Comment:
See https://docs.pytest.org/en/7.0.x/how-to/capture-warnings.html#additional-use-cases-of-warnings-in-tests

Reply:
You can use this utility we have:

```
from tests.helpers.utils import no_warning_call

with no_warning_call(match="..."):
    ...
```